gui/render_text.cpp

```diff
@@ -17,18 +17,19 @@ void render_text(
 	// helper variables
 	int textW = xEnd - xStart;
 	int textH = yEnd - yStart;
+	int lineHeight = fontSize * 1.4; // determine space between lines
```

Comment:
We could put stuff like this in a config file so that we can modify the graphics by fiddling with the config instead of roaming through the source... (Same would go for colors and so on)
But I would consider that an improvement for later (probably never?)